### ISSUE TITLE
TextInput caret position with center alignment and prompt text

### ISSUE THREAD

**Issue by timoisalive** [BowlerHatLLC/feathers#1498]
Another enhancement request for the text input would be to align the caret with the left edge of the prompt text when both the editor and prompt are center aligned.

At the moment the caret is blinking in the middle of the prompt text.

**Comment by joshtynjala** [comment]
It seems a little strange that the caret would jump from left of center to right of center once you a type a character. That may be the correct behavior here, though.

**Comment by timoisalive** [comment]
True, you would think it's strange, but it actually works quite naturally.
